src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.interop.probing.caller.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import it.pagopa.interop.probing.caller.util.EserviceTechnology;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class EserviceContent {
+
+  @JsonProperty("id")
+  private String id;
```

Comment:
String or UUID?

Reply:
is right string, is the identifier of the eservices table

Reply:
changed in eserviceRecordId to disambiguate it